### PR TITLE
Screens added 

### PR DESCRIPTION
**Pull request by msdeshmukh009**
### Pages Created
- Product Listing
- Cart
- Wishlist
- Authentication - Signin, Signup
### Preview
pages

### CODE REVIEW

File: src/components/wishlist-card/WishlistCard.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <div className="price">
+          <span className="card-sub-heading text-md">₹ {price} </span>
+          <span className="text-line-through text-xs text-gray">₹ {discountedPrice}</span>
+        </div>
+      </div>
+      <div className="card-cta-vertical">
+        <button className="btn btn-primary block-btn">
+          <span className="btn-icon">
+            <i className="fas fa-shopping-cart"></i>
+          </span>{" "}
```
Comment (gopalbbharadva) [inline src/components/wishlist-card/WishlistCard.jsx]: You can remove these extra spaces.

Comment (gopalbbharadva) [reply src/components/wishlist-card/WishlistCard.jsx]: Everything looks fine.

Comment (msdeshmukh009) [reply src/components/wishlist-card/WishlistCard.jsx]: Removing it now.

**Review by gopalbbharadva — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -11,6 +10,8 @@ const App = () => {
         <Route path="/products" element={<ProductListing />} />
```
Comment (poornima-alugubelly) [inline src/App.js:11]: hey u can make a separate file for routes, it is better to keep App as clean as possible

**Review by poornima-alugubelly — APPROVED** [review]

File: src/App.js (hunk 2 above)
Comment (msdeshmukh009) [reply src/App.js:11]: Ok, I will do that. Thanks!!

**Pull request merged**